tests/acceptance/features/coreApiShareReshareToShares3/reShareWithExpiryDate.feature

```diff
@@ -62,9 +62,6 @@ Feature: resharing a resource with an expiration date
 
   Scenario Outline: resharing with user using the sharing API with expire days set and combinations of default/enforce expire date enabled
```

Comment:
Please, rename the scenario. And also adhere to https://github.com/owncloud/QA/issues/804
```suggestion
  Scenario Outline: resharing with user using the sharing API with expire days set and combinations of default/enforce expire date enabled
```